Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Telemetry.swift

```diff
@@ -80,6 +108,40 @@ public class Telemetry {
         self.scheduler = scheduler
     }
     
+    // MARK: - Public functions
+    public func isInAir() -> Single<Bool> {
+        return Single<Bool>.create { single in
+            let inAirRequest = Dronecore_Rpc_Telemetry_SubscribeInAirRequest()
+            do {
+                let inAirResponse = try self.service.subscribeinair(inAirRequest, completion: nil)
+                while let response = try? inAirResponse.receive() {
```

Comment:
I am not sure what this will do, but I believe it will complete the single after the first event, and leak memory after that. `SubscribeInAir` returning a stream, we should have an `InAirObservable` instead of an `isInAir` single.